Common/SimConfig/src/ConfigurableParam.cxx

```diff
@@ -130,6 +130,7 @@ void ConfigurableParam::updateFromString(std::string configstring)
   if (!sIsFullyInitialized) {
     initialize();
   }
+  configstring.erase(std::remove(configstring.begin(), configstring.end(), ' '), configstring.end()); // remove spaces
```

Comment:
ok. Just a slight doubt: What about spaces within strings?
example: `param.name='Hello world'`

Reply:
It is a very valid doubt. One might question what could be the use of such strings, on the other hand I would agree that this calls for a proper parsing of strings when within single quotes to avoid the removal of spaces. Will fix it

Reply:
hi @sawenzel ,
I believe the best solution for the issue is to deal with the spaces while looping over the tokens:
- if it is a key --> remove all spaces
- if it is a value --> remove leading and trailing spaces
- if if is a value and it is not a string (value not within quotes) --> remove all spaces

Reply:
or possibly, to simplify the code and perhaps allow string-with-spaces key type (uhm...) better do regardless of the key/value type of a given token
- if it is a string, remove only leading and trailing spaces
- else remove all spaces